CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ with Swift Package Manager on Linux. When contributing code changes, please
 ensure that all three supported build methods continue to work and pass tests.
 
 ```shell
-$ xcodebuild -scheme swiftlint test
+$ xcodebuild -scheme swiftlint test -destination 'platform=macOS'
```

Comment:
Or is the idea that the instructions implicitly intend for the user to add a destination?

Reply:
I think it's fine to specify macOS explicitly. Other platforms don't make sense here.